css/src/components/hero.scss

```diff
@@ -0,0 +1,153 @@
+$hero-padding-xs: $layout-4 !default;
+$hero-padding-sm: $layout-5 !default;
+$hero-padding-default: $layout-6 !default;
+$hero-padding-lg: $layout-7 !default;
+
+$hero-min-height-xs: 200px !default;
+$hero-min-height-sm: 300px !default;
+$hero-min-height-default: 400px !default;
+$hero-min-height-lg: 500px !default;
+
+$hero-content-width: 45% !default;
+$hero-content-fluid-width: 45% !default;
+$hero-image-width: calc((100% - 2 * $layout-gap) * 0.55) !default;
+$hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !default;
+
+.hero {
+	@extend %layout-gap;
+
+	display: flex;
+	position: relative;
+	flex-direction: column;
+	align-items: stretch;
+	justify-content: space-between;
+
+	.hero-content {
```

Comment:
Any thoughts on vertically center the content of `hero-content` by default and the user can change it if they want to?  
![image](https://user-images.githubusercontent.com/94572161/181620062-beb823fc-e7c4-47ae-be94-9b28fdd9becb.png)

Reply:
That's what bothers me now after the bottom gradient is gone. Processing

Reply:
@wibjorn any thoughts on this?

Reply:
Looks like that is not intended to me

<img width="620" alt="image" src="https://user-images.githubusercontent.com/30843002/181647907-53947d55-ca32-439a-b165-489b98b67bb9.png">

Reply:
Looks like that is not intended based on the Figma

Reply:
Keeping as is for now